test/faker/geohash_test.exs

```diff
@@ -0,0 +1,7 @@
+defmodule GeohashTest do
+    use ExUnit.Case, async: true
+  
+    test "geohash/0" do
+      assert is_binary(Faker.Geohash.geohash)
+    end
+  end
```

Comment:
Something wrong with indentation here. Also, please you the final new line in the files, you can find out more about it in this answer: https://unix.stackexchange.com/a/18789.